### DISCUSSION TITLE
404 error when trying to download

### DISCUSSION THREAD

**Discussion by chuck-gulotta** [martinet101/ElevenClock#343] [General]
When I am clicking on latest release to download eleven clock, I am getting a 404 error.  I am trying to download version 2.8, which is the latest release version.

**Comment by chuck-gulotta** [comment]
I guess it was just an internet issue, I was able to download now.

**Comment by pfeiffma** [comment]
This issue is occurring for me as well. Whether I attempt to download it from releases page or home page, continually get a 404 error.

**Comment by marticliment** [comment]
I guess there may have been issues with github releases server. If this continues i'llopen a support ticket

**Comment by Kortex49** [comment]
Same issue here. E404 on github when trying to download the latest release.

**Reply by marticliment** [reply]
In which country are you?

**Reply by pfeiffma** [reply]
I'm not sure about Kortex, but I am in the US. Also getting the 404 error right now when trying to download.

**Reply by Kortex49** [reply]
UK for me

**Reply by Kortex49** [reply]
Managed to download the latest release from a 3rd party site. All releases I've tried downloading so far today ended up with a 404 error on github.

**Reply by marticliment** [reply]
The Microsoft store install also fails?

**Reply by Kortex49** [reply]
I didn't know it was available on the Microsoft store. No reason it would not have worked though.

**Comment by PescheHelfer** [comment]
Getting the same again. I t worked in the past once. I wonder if this is a github issue, but I don't know what one can do about it (other than finding other download sources)

**Comment by marticliment** [comment]
Does the built-in updater work?

**Reply by PescheHelfer** [reply]
That worked nicely, thx!
